docker/init.cmd

```diff
@@ -0,0 +1,13 @@
+:: Change directory to root of the repo
+cd /D "%~dp0\.."
```

Comment:
I haven't tested this line as I don't have a windows setup. If someone could test this on Windows I'd be grateful.

Reply:
will do

Reply:
> PS C:\Users\james\RubymineProjects\otwarchive\docker> .\init.cmd
> 
> C:\Users\james\RubymineProjects\otwarchive\docker>cd /D "C:\Users\james\RubymineProjects\otwarchive\docker\\.."
> 
> C:\Users\james\RubymineProjects\otwarchive>copy docker\database.yml config\database.yml
>         1 file(s) copied.
> 
> C:\Users\james\RubymineProjects\otwarchive>copy docker\redis.yml config\redis.yml
>         1 file(s) copied.
> 
> C:\Users\james\RubymineProjects\otwarchive>copy docker\local.yml config\local.yml
>         1 file(s) copied.
> 
> C:\Users\james\RubymineProjects\otwarchive>docker-compose up -d
> Creating network "otwarchive_default" with the default driver
> Creating otwa_mc    ... done
> Creating otwa_es    ... done
> Creating otwa_redis ... done
> Creating otwa_db    ... done
> Creating otwa_web   ... done
> 
> C:\Users\james\RubymineProjects\otwarchive>timeout 60
> 
> Waiting for  0 seconds, press a key to continue ...
> Starting otwa_redis ... done
> Starting otwa_mc    ... done
> Starting otwa_es    ... done
> Dropped database 'otwarchive_development'
> Dropped database 'otwarchive_test'
> Created database 'otwarchive_development'
> Created database 'otwarchive_test'
> -- create_table("abuse_reports", {:id=>:integer, :force=>:cascade, :options=>"ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci ROW_FORMAT=DYNAMIC"})

Reply:
That looks good. Yeah I just wasn't sure the working directory would change correctly so the file copies would fail, but LGTM there